docs/components/form.md

```diff
@@ -105,16 +107,19 @@ This is recommended over `<form />`. When the `action` prop is omitted, `<Form>`
 
 **Related APIs:**
 
-- [`useNavigation`][usenavigation]
-- [`useActionData`][useactiondata]
-- [`useSubmit`][usesubmit]
+- [`useActionData`][use-action-data]
+- [`useNavigation`][use-navigation]
+- [`useSubmit`][use-submit]
+
+[use-navigation]: ../hooks/use-navigation
+
+<!-- page doesn't exist anymore -->
 
-[index query param]: ../guides/routing#what-is-the-index-query-param
-[usenavigation]: ../hooks/use-navigation
-[useactiondata]: ../hooks/use-action-data
-[usesubmit]: ../hooks/use-submit
+[index-query-param]: ../guides/routing#what-is-the-index-query-param
```

Comment:
This section (https://remix.run/docs/en/dev/guides/routing#what-is-the-index-query-param) doesn't seem to exist anymore

Reply:
For now do you want to just copy the markdown file from https://reactrouter.com/en/main/guides/index-search-param over into the Remix repo under guides and link to it?